### ISSUE TITLE
Tiny-ImageNet only Acc@1=62.4500

### ISSUE THREAD

**Issue by Longchentong** [kschwethelm/HyperbolicCV#3]
python code/classification/train.py -c classification/config/L-ResNet18.txt --device "cuda:5"

Results: Loss=2.0732, Acc@1=62.4500, Acc@5=79.4100 

I totally keep the same setting with the code without any other change
CIFAR-100, CIFAR-10 can get ideal result in the paper
but Tiny-ImageNet only reaches Acc@1=62.4500

Should I change some setting for Tiny-ImageNet?

```
# python code/classification/train.py -c classification/config/L-ResNet18.txt

# Output settings
exp_name = L-ResNet18
#output_dir = classification/output

# General settings
device = cuda:0
dtype = float32
seed = 1
#load_checkpoint = classification/output/L-ResNet18.pth

# General training hyperparameters
num_epochs = 200
batch_size = 128
lr = 1e-1
weight_decay = 5e-4
optimizer = RiemannianSGD
use_lr_scheduler = True
lr_scheduler_milestones = [60,120,160]
lr_scheduler_gamma = 0.2

# General validation/testing hyperparameters
batch_size_test = 128

# Model selection
num_layers = 18
embedding_dim = 512
encoder_manifold = lorentz
decoder_manifold = lorentz

# Manifold settings
# learn_k = True
encoder_k = 1.0
decoder_k = 1.0

# Dataset settings
dataset = CIFAR-100 # CIFAR-10 or Tiny-ImageNet
```

**Comment by Longchentong** [comment]
Hi

**Comment by inboxedshoe** [comment]
Hey, it's been a while since the project, but I'll try to go through the old files again, find the correct settings, and send them. I believe they were slightly different from CIFAR defaults.

**Comment by Longchentong** [comment]
> Hey, it's been a while since the project, but I'll try to go through the old files again, find the correct settings, and send them. I believe they were slightly different from CIFAR defaults.

Thank you, looking forward to your reply! Really need this setting.🌹
